### PR TITLE
Add perspective package

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by timkpaine** [comment]
@texodus @neilslinger please confirm you want to be listed as maintainer

**Comment by timkpaine** [comment]
https://github.com/finos/perspective/pull/930 for license

**Comment by timkpaine** [comment]
https://github.com/finos/perspective/pull/927 to remove yarn and pass tests

File: recipes/perspective/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - six >=1.11.0
+    - traitlets >=4.3.2
+
+test:
+  imports:
+    - perspective
+
+about:
+  home: http://github.com/finos/perspective
+  license: Apache 2.0
```
Comment (marcelotrevisani) [inline recipes/perspective/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/perspective/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - python
+    - pip
+    - pybind11
+    - boost
+    - numpy >=1.13.1
+    - pyarrow ==0.15.1
+  run:
+    - python
+    - ipywidgets >=7.5.1
+    - numpy >=1.13.1
```
Comment (marcelotrevisani) [inline recipes/perspective/meta.yaml]:
```suggestion
    - {{ pin_compatible('numpy') }}
```

**Comment by timkpaine** [comment]
thanks @marcelotrevisani ! just waiting on a few PRs, @texodus is getting around to them this week. then i'll bump the version and we'll be good to go

**Comment by texodus** [comment]
> I am a maintainer and want to be listed.  

Like that?

**Comment by neilslinger** [comment]
Yes - please list me as a maintainer too

**Comment by timkpaine** [comment]
0.4.4 will remove the need for `npm` or `yarn`, has better VS/VC support (defaults to VC14.1/VS2017, with support for VC14/VS2015) and will bundle the `LICENSE`, which should clear up the tests. I will bump this recipe when the release is done.

File: recipes/perspective/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - python
+    - ipywidgets >=7.5.1
+    - {{ pin_compatible('numpy') }}
+    - python-dateutil >=2.8.0
+    - six >=1.11.0
+    - traitlets >=4.3.2
+
+test:
+  imports:
+    - perspective
```
Comment (ceball) [inline recipes/perspective/meta.yaml]:
```suggestion
    - perspective
    - perspective.table
```

@timkpaine what about adding an import to catch missing or bad c++ bindings? Not sure which is the most appropriate import (in terms of coverage and being stable).

**Comment by timkpaine** [comment]
@conda-forge/help-python all set for now

**Pull request merged**